website/site/content/docs/add-to-your-site.md

```diff
@@ -92,7 +92,7 @@ These lines specify your backend protocol and your publication branch. Git Gatew
 
 ### Editorial Workflow
 
-By default, saving a post in the CMS interface will push a commit directly to the publication branch specified in `backend`. However, you also have the option to enable the [Editorial Workflow](https://www.netlifycms.org/docs/configuration-options/#publish-mode), which adds an interface for drafting, reviewing, and approving posts. To do this, add the following line to your `config.yml`:
+By default, saving a post in the CMS interface will push a commit directly to the publication branch specified in `backend`. However, you also have the option to enable the [Editorial Workflow](https://www.netlifycms.org/docs/configuration-options/#publish-mode), which adds an interface for drafting, reviewing, and approving posts. To do this, add the following line to your Netlify CMS `config.yml`:
```

Comment:
We should now mention that editorial workflow is only currently support by GitHub and Git Gateway over GitHub.